packages/expo-notifications/src/NotificationsEmitter.ts

```diff
@@ -121,3 +125,17 @@ export async function getLastNotificationResponseAsync(): Promise<NotificationRe
   const mappedResponse = response ? mapNotificationResponse(response) : response;
   return mappedResponse;
 }
+
+/**
+ * Clears the notification response that was received most recently. May be used in cases
+ * where the app selects a route based on the notification response, and it is undesirable
+ * to continue to select the route after the response has already been handled.
```

Comment:
```suggestion
 * Clears the notification response that was received most recently. May be used 
 * when an app selects a route based on the notification response, and it is undesirable
 * to continue selecting the route after the response has already been handled.
```

A suggestion to rephrase the second sentence to improve readability.